### ISSUE TITLE
Tactile hallucination

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6777]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
The false perception of tactile sensory input that creates a hallucinatory sensation of physical contact with an imaginary object

Comment:
There are many different types of tactile hallucination. A common one is formication, the sensation like insects crawling over the skin (derived from the Latin word for ant, formica).

**Parent term (use hpo.jax.org/app)**
Hallucinations HP:0000738

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
